src/parameters.toml

```diff
@@ -491,6 +491,50 @@ alias = "k_thrshld_stpnss"
 value = 2.0
 type = "float"
 
+[SB2001_Long_collection_kernel_coeff_kc]
+alias = "kc_SB2001"
+value = 9.44e9
+type = "float"
```

Comment:
Could you add a short description of each parameter that mentions the units of each parameter and the paper they are coming from? Something like [this](https://github.com/CliMA/CLIMAParameters.jl/blob/26bde4da7633002e3cab7f1e57ebbcf41fbd9f30/src/parameters.toml#L595-L599)?

I know this is not the prevailing pattern here, but I'm working on fixing it.